department-of-veterans-affairs/vets-website
# 76117b applicant ohi upload screens

## Summary

- Adds another two conditional file upload screens (other health insurance and 10-7959c)
- Updated submit transformer with new file upload fields

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/76117

## Testing done

- Manual testing
- Ran yarn `test:coverage-app ivc-champva/10-10D` to verify existing tests run and pass (more planned in an upcoming PR)

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Screen 1 |
| ------- | ------ |
| Desktop |![Screenshot 2024-02-23 at 12 00 05](https://github.com/department-of-veterans-affairs/vets-website/assets/18408628/661149ef-5058-4c6a-8136-de337d459daf)|

## What areas of the site does it impact?

This form only

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

NA
